File: Exercises/101-109 - Small Problems/easy_2/how_old_is_teddy.rb
# Build a program that randomly generates and prints Teddy's age.
# To get the age, you should generate a random number between 20 and 200.
# Modify this program to ask for a name,
# and then print the age for that person. For an extra challenge, use "Teddy" as the name if no name is entered.

puts 'Please enter a name:'
name = gets.chomp
name = 'Teddy' if name == ''
puts "#{name} is #{rand(20..200)} years old!"

File: Exercises/210 - Small Problems/easy-3/how-old-is-teddy.js
/*
Build a program that randomly generates and logs Teddy's age to the console. Have the age be a random number between 20
and 200 (inclusive).

Example Output

Teddy is 69 years old!
*/

var age = Math.floor(Math.random() * (200 - 20 + 1)) + 20;

console.log('Teddy is ' + age + ' years old!');